realsense2_camera/tools/frame_latency/frame_latency.h

```diff
@@ -0,0 +1,29 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved
+
```

Comment:
Changes due to Merge with ROS2-beta branch,
which includes 3 pushes: 
-ROS2-beta Release (4.0.3) commits by SamerKhshiboun
-PR2288 by Nir-Az
-PR2279 by Nir-Az